ui/admin/app/routes/scopes/scope/aliases/index.js

```diff
@@ -123,7 +123,6 @@ export default class ScopesScopeAliasesIndexRoute extends Route {
       'alias',
       {
         scope_id,
-        query: { filters: { scope_id: [{ equals: scope_id }] } },
```

Comment:
Removed this as this will actually end up slowing down the query and this is only to check for existence of data.